### ISSUE TITLE
cannot find package gokogiri

### ISSUE THREAD

**Issue by jeena** [arturovm/pond#9]
I've installed the dependencies:

```
➜ jeena@Lala pond (master) go get -u github.com/bmizerany/pat
➜ jeena@Lala pond (master) go get -u github.com/mattn/go-sqlite3
➜ jeena@Lala pond (master) go get -u github.com/robfig/cron
➜ jeena@Lala pond (master) go get -u code.google.com/p/go.crypto/bcrypt
```

But when I try to build I get this error:

```
➜ jeena@Lala pond (master) go build -o pond *.go
articles_download.go:6:2: cannot find package "github.com/moovweb/gokogiri" in any of:
    /usr/local/Cellar/go/1.2/libexec/src/pkg/github.com/moovweb/gokogiri (from $GOROOT)
    /Users/jeena/go/src/github.com/moovweb/gokogiri (from $GOPATH)
articles_download.go:7:2: cannot find package "github.com/moovweb/gokogiri/xml" in any of:
    /usr/local/Cellar/go/1.2/libexec/src/pkg/github.com/moovweb/gokogiri/xml (from $GOROOT)
    /Users/jeena/go/src/github.com/moovweb/gokogiri/xml (from $GOPATH)
```

**Comment by jeena** [comment]
I had to install `go get github.com/moovweb/gokogiri` but to do that I got a `Package libxml-2.0 was not found in the pkg-config search path.`. I had to do: `brew install libxml2` and then `brew link --force libxml2` after that `go get github.com/moovweb/gokogiri` worked and I was able to build.
Perhaps you could add `gokogiri` to the dependencies?

**Comment by arturovm** [comment]
Done

**Comment by jeena** [comment]
It would be cool if you could close issues with "fixes #9" for example in your commit messages, that way github will link this issue to the right commit and it is easier to check what you've done ;)
